pkg/util/validation/limits.go

```diff
@@ -66,25 +67,25 @@ func (e LimitError) Error() string {
-
+	RequestRate                           float64             `yaml:"request_rate" json:"request_rate"`
+	RequestBurstSize                      int                 `yaml:"request_burst_size" json:"request_burst_size"`
+	IngestionRate                         float64             `yaml:"ingestion_rate" json:"ingestion_rate"`
+	IngestionBurstSize                    int                 `yaml:"ingestion_burst_size" json:"ingestion_burst_size"`
+	AcceptHASamples                       bool                `yaml:"accept_ha_samples" json:"accept_ha_samples"`
+	HAClusterLabel                        string              `yaml:"ha_cluster_label" json:"ha_cluster_label"`
+	HAReplicaLabel                        string              `yaml:"ha_replica_label" json:"ha_replica_label"`
+	HAMaxClusters                         int                 `yaml:"ha_max_clusters" json:"ha_max_clusters"`
+	DropLabels                            flagext.StringSlice `yaml:"drop_labels" json:"drop_labels" category:"advanced"`
+	MaxLabelNameLength                    int                 `yaml:"max_label_name_length" json:"max_label_name_length"`
+	MaxLabelValueLength                   int                 `yaml:"max_label_value_length" json:"max_label_value_length"`
+	MaxLabelNamesPerSeries                int                 `yaml:"max_label_names_per_series" json:"max_label_names_per_series"`
+	MaxMetadataLength                     int                 `yaml:"max_metadata_length" json:"max_metadata_length"`
+	MaxNativeHistogramBuckets             int                 `yaml:"max_native_histogram_buckets" json:"max_native_histogram_buckets"`
+	CreationGracePeriod                   model.Duration      `yaml:"creation_grace_period" json:"creation_grace_period" category:"advanced"`
+	EnforceMetadataMetricName             bool                `yaml:"enforce_metadata_metric_name" json:"enforce_metadata_metric_name" category:"advanced"`
+	IngestionTenantShardSize              int                 `yaml:"ingestion_tenant_shard_size" json:"ingestion_tenant_shard_size"`
+	MetricRelabelConfigs                  []*relabel.Config   `yaml:"metric_relabel_configs,omitempty" json:"metric_relabel_configs,omitempty" doc:"nocli|description=List of metric relabel configurations. Note that in most situations, it is more effective to use metrics relabeling directly in the Prometheus server, e.g. remote_write.write_relabel_configs. Labels available during the relabeling phase and cleaned afterwards: __meta_tenant_id" category:"experimental"`
+	EnableServiceOverloadErrorOnRateLimit bool                `yaml:"enable_service_overload_error_on_rate_limit" json:"enable_service_overload_error_on_rate_limit" category:"experimental"`
```

Comment:
```suggestion
	ServiceOverloadStatusCodeOnRateLimitEnabled bool                `yaml:"service_overload_status_code_on_rate_limit_enabled" json:"service_overload_status_code_on_rate_limit_enabled" category:"experimental"`
```

We are trying to use "enabled" as a suffix for new flags. Also changed "error" to status code.